app/models/task.py

```diff
@@ -1,5 +1,27 @@
-from app import db
-
+from app import db 
+from app.models.goal import Goal
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.id'), nullable=True)
+
+    def response_dict(self):
```

Comment:
I _really_ like that you named this function `response_dict`, I think it's a great, accurate, and more usable name.